### ISSUE TITLE
Resolve module attribute through use

### ISSUE THREAD

**Issue by odk211** [KronicDeth/intellij-elixir#1292]
Hi, related to https://github.com/KronicDeth/intellij-elixir/pull/1255,  I hope, module attribute defined in `__using__` macro is resolved too. 
thanks,

**Comment by KronicDeth** [comment]
No, module attributes use a different `PsiReference` subclass, so they've have a different `getVariants` method and would need to be updated separately to resolve through the new `use` logic.  IIRC, right now they only look in the current module and it isn't `Module.register_attribute` aware either.

**Comment by KronicDeth** [comment]
Thanks for an example library to test against.  Any chance you have a public repo using `yacto`, so I can just clone a full project?

**Comment by odk211** [comment]
@KronicDeth yacto has basic sample app in test, so you can use it :)
https://github.com/gumi/yacto/tree/master/test/apps/custom_table_name

When open custom_table_name.ex in intellij-elixir, `@auto_source` is not resolved, (it is defined by  [`use Yacto.Schema`](https://github.com/gumi/yacto/blob/master/lib/yacto/schema.ex#L38) 
<img width="556" alt="2018-10-10 0 19 10" src="https://user-images.githubusercontent.com/1321120/46679807-b0aecc00-cc22-11e8-9827-47d646f68a8c.png">

Thanks!

**Comment by chess4ever** [comment]
It would be great if this could be fixed.
